### ISSUE TITLE
jQuery - GetProperty

### ISSUE THREAD

**Issue by jimmygilles** [nikhilk/scriptsharp#106]
Hello,

I found a bug with the jQuery Wrapper.

The method jQueryObject.GetProperty in the C# DLL returns always a string, this seems correct because in the documentation of jQuery, the return value of the "prop" method is a String (http://api.jquery.com/prop/)

But, it seems the documentation of jQuery is not up to date because if I get the "checked" property of a CheckboxElement, jQuery returns a Boolean.

``` C#
Script.Alert(jQuery.Select("... a checkbox element...").GetProperty("checked").GetType().ToString());
```

displays

``` javascript
function Boolean() {
    [native code]
}
```

Thank you Nikhilk.

**Comment by jimmygilles** [comment]
If necessary, there is a simple workaround:

``` C#
jQuery.Select("... a checkbox element...")[0].As<CheckBoxElement>().Checked
```

**Comment by nikhilk** [comment]
Fix included in 0.7.4.0
